nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -243,6 +246,7 @@ public StandardProcessGroup(final String id, final ControllerServiceProvider ser
         this.defaultFlowFileExpiration = new AtomicReference<>();
         this.defaultBackPressureObjectThreshold = new AtomicReference<>();
         this.defaultBackPressureDataSizeThreshold = new AtomicReference<>();
+        this.logFileSuffix = getName();
```

Comment:
We should not default to creating a separate log file per PG. We should default to this being `null`.